home.php

```diff
@@ -0,0 +1,108 @@
+<?php
+  session_start();
+  ini_set("display_error", "1");
+
+ include("session_check.php");
```

Comment:
almost every where include is used and not require_once.. is there any reason?
try to see why to use require_once and fix everywhere in the code.

also use single quote everywhere if not using any php code or variable or anything to parse.